canonical-web-and-design/ubuntu.com
# Ensure forms in blog posts inherit URL UTM params

## Done

* While most forms are able to inherit UTMs from the context, forms in blog post have no mechanism for it. This branch introduces a global mechanism to pass URL UTM params to their respective marketo fields when present on the page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit [/blog/canonical-at-techweek-frankfurt?utm_source=foo&utm_campaign=bar](https://ubuntu-com-canonical-web-and-design-pr-6108.run.demo.haus/blog/canonical-at-techweek-frankfurt?utm_source=foo&utm_campaign=bar)
- Inspect the source of the form and ensure the utm_source and utm_campaign fields contain the right values
